### ISSUE TITLE
Support Rails 6

### ISSUE THREAD

**Issue by rickychilcott** [modosc/cloudflare-rails#8]
```bash
Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 6.1.0.alpha)

  In Gemfile:
    rails

    cloudflare-rails (>= 0.2) was resolved to 0.4.0, which depends on
      rails (~> 5.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

Assuming it's OK, I think we can loosen up the rails requirement to be `> 5.0` instead of `~> 5.0)` for version 0.2 of this gem.

**Comment by modosc** [comment]
i haven't tested it with 6.0 yet - there were breaking changes between 4.x and 5.x so it's fair to assume the same will happen here. i'll try to get to this sometime this week (or a pr would be welcome)

**Comment by rickychilcott** [comment]
I just did a quick test locally by changing the gemspec to allow it to at least be included (i.e. change the line `spec.add_dependency "rails", "~> 6.0.0.rc1"` and it seems to at least not raise any exceptions locally. How do you test/develop locally to ensure everything looks ok? 

I did a little puts debugging and everything appears to fetch IPs correctly and to patch the rack application correctly. So maybe there weren't any changes in Rails 6 to cause any changes here.

**Comment by modosc** [comment]
ok, added [specs and relaxed the rails version](https://github.com/modosc/cloudflare-rails/commit/906559589fa0a049c0f7e0645eee66e1c045da6b), released as `0.6.0`

**Comment by rickychilcott** [comment]
Cool. Glad you’ve got appraisal in place now too. Cheers! 🙏🥂
